### DISCUSSION TITLE
Here is the French translation (based upon latest en-US) for "TimVer_0.10.0"

### DISCUSSION THREAD

**Discussion by logounet** [Timthreetwelve/TimVer#79] [General]
Hello,

Here is the French translation (based upon latest en-US) for "TimVer_0.10.0"

See attached file

[Strings.fr-FR.xaml.zip](https://github.com/user-attachments/files/18135127/Strings.fr-FR.xaml.zip)

Best regards

**Comment by Timthreetwelve** [comment]
Thank you for the updated translation! It will be in the next update.

Tim
